File: Swift/Easy/Jewels-And-Stones.swift
class Solution {
    func numJewelsInStones(_ J: String, _ S: String) -> Int {
        
        var myDict = [Character: Int]()
        var count = 0
        for jwel in J {
            if myDict[jwel] == nil {
                myDict[jwel] = 1
            }
        }
        
        for stone in S {
            if myDict[stone] != nil {
                count += 1
            }
        }
        return count
    }
}

File: Python/Easy/Jewels-And-Stones.py
class Solution:
    def numJewelsInStones(self, J: str, S: str) -> int:
        
        myDict = {}
        count = 0
        for jwel in J:
            if jwel not in myDict:
                myDict[jwel] = 1
        for stone in S:
            if stone in myDict:
                count += 1
        return count
